src/texture.lisp

```diff
@@ -179,3 +179,13 @@
 
 (default-free rl-render-texture)
 (default-free-c claylib/ll:render-texture unload-render-texture)
+
+;; TODO this is messed up
+(defmethod draw-object ((obj rl-render-texture))
+  (claylib/ll:draw-texture-rec (c-struct obj)
+                               (c-struct (make-rectangle
+                                          0 0
+                                          (width (texture obj)) (height (texture obj))
+                                          +white+))
+                               (c-struct (make-vector2 0 0))
+                               (c-struct (tint (texture obj)))))
```

Comment:
It feels like I should be making a more claylib-esque `render-texture` class and defining `draw-object` on that but I got a little rattled trying to reinterpret the texture hierarchy.

Reply:
Consing aside, you shouldn't be trying to draw a render texture at all. You might need to draw a texture that is _backing_ a render texture, though. We do this in core example 26.

Reply:
Ah, I got it now, thanks!